src/main/java/net/mcreator/ui/ide/mcfunction/MinecraftCommandsTokenMaker.java

```diff
@@ -1052,7 +1052,7 @@ public MinecraftCommandsTokenMaker(java.io.InputStream in) {
 	/**

```

Comment:
This file is autogenerated and should be left as it is, in case in the future, token maker is changed